src/adb/logcat.js

```diff
@@ -0,0 +1,10 @@
+let PassThrough = require('stream').PassThrough;
+let spawn = require('child_process').spawn;
+
+module.exports = function logcat() {
+  let proc = spawn('adb', ['logcat']);
```

Comment:
If you want to skip that part for now, that's fine, but Raptor won't be able to use it until it can target a device. Unfortunately in Jenkins automation there are multiple devices on a node, so it would block us from using it until there was that support.